drracket-test/tests/drracket/private/module-lang-test-utils.rkt

```diff
@@ -146,13 +146,10 @@
          (let loop ([snip (send interactions-text find-first-snip)])
            (cond
              [(not snip) '()]
-             [else
-              (cond
-                [(method-in-interface? 'get-stacks (object-interface snip))
-                 (define-values (s1 s2) (send snip get-stacks))
-                 (list* s1 s2 (loop (send snip next)))]
-                [else
-                 (loop (send snip next))])])))))
+             [(method-in-interface? 'get-stacks (object-interface snip))
+              (define-values (s1 s2) (send snip get-stacks))
+              (list* s1 s2 (loop (send snip next)))]
+             [else (loop (send snip next))])))))
```

Comment:
This one is not a good change. This is going against the data definition. That is, the outer cond is part of the processing of `snip` (there are two cases for the input and, in the second case of the input, I want to do a conditional thing).